src/modules/keyboard/KeyboardPage.ui

```diff
@@ -117,6 +117,30 @@ SPDX-License-Identifier: GPL-3.0-or-later
      </item>
     </layout>
    </item>
+   <item>
```

Comment:
I think you can fit this next to the keyboard-type drop-down, instead, to save some vertical space.